R/map.R

```diff
@@ -140,63 +135,77 @@
 #' @examplesIf interactive() && requireNamespace("mirai", quietly = TRUE) && requireNamespace("carrier", quietly = TRUE)
 #' # Run in interactive sessions only as spawns additional processes
 #'
-#' # To use parallelized map, set daemons (number of parallel processes) first:
+#' # To use parallelized map:
+#' # 1. Set daemons (number of parallel processes) first:
 #' mirai::daemons(2)
 #'
-#' mtcars |> map_dbl(sum, .parallel = TRUE)
+#' # 2. Wrap .f with in_parallel():
+#' mtcars |> map_dbl(in_parallel(\(x) sum(x)))
 #'
+#' # Note that functions from packages should be fully qualified with `pkg::`
```

Comment:
Or load with `library()`? (Or is that not ready yet?)

Reply:
Will update once PRs at carrier are merged.

Reply:
Now updated as part of [ccf95f3](https://github.com/tidyverse/purrr/pull/1185/commits/ccf95f37a9abb8774f96ef5c2357bb2e5245a843).